web/src/AnalyticsNudge.tsx

```diff
@@ -14,7 +27,12 @@ class AnalyticsNudge extends Component {
           .)
         </div>
         <div className="AnalyticsNudge-buttons">
-          <button>Yes!</button> <button>Nope!</button>
+          <button onClick={() => AnalyticsNudge.analyticsOpt(true)}>
+            Yes!
+          </button>{" "}
```

Comment:
Hmm, I'm not sure this whitespace is the way to do what you want, but I defer to @yuindustries.

Reply:
I think Prettier did this without my consent :-/

Reply:
Oh weird, I trust it then.